docs/api-reference/decorators.md

```diff
@@ -544,6 +545,54 @@ class KafkaExample{
 }
 ```
 
+### Scheduled Workflow Decorators
+
+#### `@Scheduled(schedulerConfig: SchedulerConfig)` {#scheduled}
+
+Runs a workflow function on a specified schedule, with guarantees such as executing exactly once per scheduled interval.
+
+By default, the workflow is executed exactly once per scheduled interval.  This means executions will be started concurrently and can overlap, and that if the application is taken down and restarted, makeup work will be performed.  Tracking of execution history is implemented using system database records.  An workflow key (consisting of the workflow function name and scheduled time) is used to deduplicate any workflows that may inadvertently be initiated by the scheduler.
```

Comment:
Here we can directly say the idempotency key of a scheduled workflow is composed of the workflow function name and scheduled time.

```suggestion
By default, the workflow is executed exactly once per scheduled interval.  This means executions might be started concurrently and can overlap, and that if the application is taken down and restarted, makeup work will be performed.  Tracking of execution history is implemented using system database records.  An workflow idempotency key (consisting of the workflow function name and scheduled time) is used to deduplicate any workflows that may inadvertently be initiated by the scheduler.
```